pandas/types/validate.py

```diff
@@ -0,0 +1,7 @@
+from pandas.types.common import is_bool
+
+def _validate_bool_type(value, arg_name):
```

Comment:
@jorisvandenbossche would you recommend making the validation function itself accept a keyworded variable-length argument? or leave as is and just rename?